server.js

```diff
@@ -0,0 +1,73 @@
+var express = require('express');
+
+import { request, abort, } from 'request-promise';
```

Comment:
I gave this server a quick spin and it looks like `request-promise` doesn't support named exports anymore. (side note - also looks like this package is deprecated).  But, as a suggestion, since `request-promise` is a CommonJS module, we can import the modules you are using with the default export like so:

```suggestion
import pkg from 'request-promise';
const { request, abort } = pkg;
```